source/libs/api.js

```diff
@@ -28,6 +32,8 @@ export default async endpoint => {
 		console.error(
 			'Refined GitHub couldn’t use GitHub’s API because the token seems to be incorrect or expired. Update it in the options.'
 		);
+	} else if (json.message === 'Not Found' && endpoint.includes('following')) {
+		// Prevent console error on profile page if user does not follow you
```

Comment:
This is too specific to the feature and should not be handled directly here. Better to add a generic option of some kind to the fetch method. Same with code on line 19-23.

Reply:
Oh duh, yeah this seemed wrong but for some reason I didn't even think about catching in the feature file.